packages/third-parties/src/types/google.ts

```diff
@@ -11,11 +13,13 @@ export type GTMParams = {
   dataLayerName?: string
   auth?: string
   preview?: string
+  nonce?: ScriptProps['nonce']
```

Comment:
We can use simple type to avoid relying on `next/script` only for types

```suggestion
  nonce?: string
```

Reply:
Happy to change, but I figured this made more sense in case `<Script>` accepts different values for `nonce` in the future - is there a downside to importing from `next/script`?

As for your top-level comment:

> can we add a test for it?

I mentioned above, I'd like to, but:

> Unfortunately, my laptop froze when trying to run non-unit tests or build, so I haven't been able to do that. I also didn't find existing unit tests I could expand on. However, I did copy the code https://github.com/mozilla/blurts-server/pull/4160, and it worked well there.

But if you have some pointers on what it would look like, I could see if I could give it a shot.